starknet_devnet/util.py

```diff
@@ -243,3 +244,26 @@ def get_fee_estimation_info(tx_fee: int, gas_price: int):
 def warn(msg: str, file=sys.stderr):
     """Log a warning"""
     print(f"\033[93m{msg}\033[0m", file=file)
+
+
+class SuppressLogger:
```

Comment:
Class names shouldn't be verbs but nouns (with possible descriptors). E.g. `SuppressedLogger` or `LogSuppressor`.